src/dmd/mtype.d

```diff
@@ -1175,7 +1175,9 @@ extern (C++) abstract class Type : RootObject
     {
         Dsymbol s = toDsymbol(sc);
         if (s)
+        {
             s.checkDeprecated(loc, sc);
+        }
```

Comment:
Think the source prefs no brackets for single-line if statements. Could also move the `Dsymbol s = toDsymbol(sc)` into the if statement.

Reply:
Done.  Thank you.